sdk/notificationhubs/notification-hubs/src/models/notification.ts

```diff
@@ -29,6 +29,21 @@ export interface NotificationCommon {
   headers?: Record<string, string | undefined>;
 }
 
+/**
+ * The common notification parameters to accept a string body or JSON body.
+ */
+export interface NotificationCommonParams {
+  /**
+   * The body for the push notification.
+   */
+  body: string | unknown;
+
+  /**
+   * The headers to include for the push notification.
+   */
+  headers?: Record<string, string | undefined>;
```

Comment:
Where are the headers coming from / why are they needed? I did not see it in the description but really only asking if it's net new (as opposed to refactoring existing behavior)

Reply:
Also what would the meaning of <string, undefined> be for a header? Don't all headers need a value?

Reply:
Changed it to `Record<string, unknown>`.  This was for extensible purposes such as the `AppleHeaders` and `WnsHeaders`.  This should get around the `undefined` issue.